infra/aws/terraform/kops-infra-ci/eks.tf

```diff
@@ -37,37 +41,25 @@ module "eks" {
     "scheduler"
   ]
 
-  manage_aws_auth_configmap = true
-
-  aws_auth_roles = [
-    {
-      # AWS role used by prow to authenticate to build clusters
-      # Please, keep it in sync with prow deployment (AWS_ROLE_ARN)
-      rolearn  = "arn:aws:iam::468814281478:role/Prow-EKS-Admin"
-      username = "arn:aws:iam::468814281478:role/Prow-EKS-Admin"
-      groups   = ["system:masters"]
-    }
-  ]
-
   cloudwatch_log_group_retention_in_days = 30
 
   cluster_addons = {
     coredns = {
-      most_recent       = true
-      resolve_conflicts = "OVERWRITE"
+      most_recent                 = true
+      resolve_conflicts_on_update = "OVERWRITE"
```

Comment:
fyi, on v20, all addons default to this value https://github.com/terraform-aws-modules/terraform-aws-eks/blob/1627231af669796669ce83e0a4672a7e6d94a0b3/main.tf#L503

Reply:
Dropped